app/templates/_package.json

```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo <% if (babel) { %>src/*.js test*.js && npm run babel <% } %>&& <% if (nyc) { %>nyc <% } %>ava"<% if (babel) { %>,
+    "babel": "babel src --out-dir=. --presets=es2015",
```

Comment:
any thought? @sindresorhus. using dist would be fine it could cover many of files. but separating configurations that it is a little bit profuse in my point of view. I have no strong taste for it